compose_rl/interfaces/base_kl_controller.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 MosaicML ComposeRL authors
+# SPDX-License-Identifier: Apache-2.0
+
+from abc import ABC, abstractmethod
+from typing import Any
+
+import torch
+
+__all__ = ['BaseKLController']
+
+
+class BaseKLController(ABC):
+    """Abstract base class for KL controller classes.
+
+    Args:
+        device: the device to run the KL controller on.
+        *args: additional arguments.
+        **kwargs: additional keyword arguments.
+    """
+
+    def __init__(self, device: str, *args: Any, **kwargs: Any):
+        del device, args, kwargs  # unsued here
```

Comment:
```suggestion
        del args, kwargs  # unsued here
        self.device = device
```

why not set device under the `super()` call